easybuild/easyconfigs/p/PaStiX/pastix-memory.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru pastix_5.2.2.22-orig/src/config/LINUX-GNU.in pastix_5.2.2.22/src/config/LINUX-GNU.in
```

Comment:
please include patch description + mention author above (and update checksum accordingly)